<!-- Launch file to move on a preprogrammed trajectory -->

<launch>

  	<arg name="arm" default="PSM1"/>

  
	<node name="home_arm"
  		pkg="irob_dvrk_automation" type="home" output="screen">
  		
  		<param name="arm" type="string" value="$(arg arm)" />

	</node>
  	
</launch>
